### PR TITLE
Clean not need ctx arguments.

### PR DESCRIPTION
**Pull request by gongweibao**
Fix part of https://github.com/PaddlePaddle/Paddle/issues/10804

### CODE REVIEW

File: paddle/fluid/operators/detail/grpc_client.cc [hunk 1]
```diff
@@ -67,8 +63,26 @@ bool RPCClient::AsyncSendVariable(const std::string& ep,
 
 void ProcGetResponse(const VarHandle& var_h,
                      const ::grpc::ByteBuffer& ret_msg) {
+  auto var = var_h.scope->FindVar(var_h.name);
+
+  platform::Place place = platform::CPUPlace();
+  if (var->IsType<framework::LoDTensor>()) {
```
Comment (jacquesqiao) [inline paddle/fluid/operators/detail/grpc_client.cc]: Add a function to get the place？

File: paddle/fluid/operators/detail/grpc_client.cc [hunk 2]
```diff
@@ -67,8 +63,26 @@ bool RPCClient::AsyncSendVariable(const std::string& ep,
...
+    place = var->Get<framework::LoDTensor>().place();
+  } else if (var->IsType<framework::SelectedRows>()) {
+    place = var->Get<framework::SelectedRows>().place();
+#ifdef PADDLE_WITH_CUDA
+  } else if (var->IsType<ncclUniqueId>()) {
```
Comment (jacquesqiao) [inline paddle/fluid/operators/detail/grpc_client.cc]: No return value for this branch.

File: paddle/fluid/operators/detail/grpc_client.h [hunk 3]
```diff
@@ -196,6 +189,8 @@ class RPCClient {
   grpc::CompletionQueue cq_;
   std::map<std::string, std::shared_ptr<grpc::Channel>> channels_;
   int64_t req_count_ = 0;
+
+  static const int64_t rpc_time_out = 600 * 1000;
```
Comment (jacquesqiao) [inline paddle/fluid/operators/detail/grpc_client.h]: rpc_time_out   ==> kRpcTimeOut
https://google.github.io/styleguide/cppguide.html#Constant_Names

File: paddle/fluid/operators/detail/grpc_client.cc (hunk 1 above)
Comment (gongweibao) [reply paddle/fluid/operators/detail/grpc_client.cc]: Done.

File: paddle/fluid/operators/detail/grpc_client.cc (hunk 2 above)
Comment (gongweibao) [reply paddle/fluid/operators/detail/grpc_client.cc]: It uses the default value. 
Done.

File: paddle/fluid/operators/detail/grpc_client.h (hunk 3 above)
Comment (gongweibao) [reply paddle/fluid/operators/detail/grpc_client.h]: Great.Done.

**Pull request closed without merging**